src/FSharpSource.targets

```diff
@@ -3,6 +3,9 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    <Choose>
     <When Condition="'$(Configuration)' == 'Debug' or '$(Configuration)' == 'Release' or '$(Configuration)' == 'VSDebug' or '$(Configuration)' == 'VSRelease' ">
+      <PropertyGroup>
+         <SkipSigning>true</SkipSigning><!-- Skip using the ,net toolset to generate signing . -->
```

Comment:
* Skip Signing turns off the OSS signing.  
* Currently we are using the desktop F# compiler to build the F# compiler, which signs perfectly well.  
* The build and deployment will need revisiting and aligning with what Roslyn does once we have something that runs and compiles.
* As well as going nuget for all of this stuff.

Too much to do, not enough fingers :-(